factory/config.go

```diff
@@ -129,14 +129,12 @@ func loadTFconfig(
 	c.FQDN = terraform.Output(g, terraformOptions, "Route53_info")
 	c.ServerIPs = strings.Split(terraform.Output(g, terraformOptions, "master_ips"), ",")
 
-	if cfg.Product == "k3s" {
-		c.Config.DataStore = terraform.GetVariableAsStringFromVarFile(g, varDir, "datastore_type")
-		if c.Config.DataStore == "external" {
-			c.Config.ExternalDb = terraform.GetVariableAsStringFromVarFile(g, varDir, "external_db")
-			c.Config.RenderedTemplate = terraform.Output(g, terraformOptions, "rendered_template")
-		} else if c.Config.DataStore == "" {
-			return nil, shared.ReturnLogError("datastore should not be empty \n%w", err)
-		}
+	c.Config.DataStore = terraform.GetVariableAsStringFromVarFile(g, varDir, "datastore_type")
```

Comment:
so its all the same config for k3s external dbs? on vars and in the terraform ?

Reply:
Yes it is the same as k3s